spec/text_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'text'
+require 'time'
+
+describe Text do
+
+  it 'sends a text' do
+    fake_message = {
+      from: '123',
+      to: '456',
+      body: 'testing my message'
+    }
+
+    expect(subject).to receive(:send_text).with(fake_message)
+    subject.send_text(fake_message)
```

Comment:
You are not testing anything of the code here.
As a rule, you should use a to_receive matcher on the subject like this:
```
expect(subject).to receive
```
or 
```
allow(subject).to receive
```
a to_receive matcher is used for interaction of the class under test **with other objects**.